client/src/plus/deep-dives/overview/app.tsx

```diff
@@ -14,9 +14,20 @@ export default function App() {
               <h1>Modern CSS in the Real World</h1>
               <h2>Featured Deep Dive</h2>
             </div>
-            <p>
-              Read a sneek peak of Rachel Andrew’s forthcoming deep dive series
-            </p>
+            <p className="author">Rachel Andrew, CSS WG Invited Expert</p>
+            <ul className="author-links">
+              <li>
+                <a href="https://twitter.com/rachelandrew" rel="external">
+                  @rachelandrew
+                </a>
+                ,
```

Comment:
This comma doesn't show up in the rendered version

Reply:
![Screenshot 2021-07-28 at 15 20 27](https://user-images.githubusercontent.com/10350960/127329315-cc8d6e1a-c362-41e1-8aee-888b52221a60.png)

It shows up for me. Unless you are referring to something else?

Reply:
oh, sorry, must have taken the wrong one, I meant the in article links 
![image](https://user-images.githubusercontent.com/44966503/127329557-9546d7f2-6f02-493b-b96a-d0cb82c6cd61.png)